docs/specification/draft/basic/transports.mdx

```diff
@@ -287,6 +288,129 @@ protocol version 2024-11-05) as follows:
        the old HTTP+SSE transport, and should use that transport for all subsequent
        communication.
 
+## WebSocket
+
+This section defines bidirectional transport of MCP over a WebSocket connection which provides better support for maintaining long-lived connections particularly in cloud environments and through intermediaries. Long-lived connections are necessary for server-initiated notifications like `notifications/resources/updated` and `notifications/*/list_changed` which enable real-time use cases.
+
+The server **MUST** provide an HTTP endpoint path that supports WebSocket upgrade HTTP request. This **SHOULD** be the same endpoint as the Streamable HTTP transport.
```

Comment:
> This **SHOULD** be the same endpoint as the Streamable HTTP transport.

This is likely to be annoying for the Python SDK, and I think for others as well.

Is there a `.well-known` we are supposed to add those endpoints, or a SEP for it?

---

In any case, I think it should't be the same.